Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM postgres:12
+FROM postgres:12@sha256:f9e8ae1fbd03f2c48d843efe7484f45913ef0283a60dec7ad0bd36a80158245b
```

Comment:
Do we really want the digest here?

Reply:
It's required to allow dependabot to bump to newer versions of the 12 container.

But I think I'll implement a cronjob instead.